asreview/webapp/src/ProjectComponents/AnalyticsComponents/ProgressRecallChart.js

```diff
@@ -175,15 +48,87 @@ export default function ProgressRecallChart(props) {
   const theme = useTheme();
   const chartRef = useRef(null);
 
+  const [anchorElPopover, setAnchorElPopover] = useState(null);
+  const [anchorElMenu, setAnchorElMenu] = useState(null);
+
+  const handlePopoverOpen = (event) => {
+    setAnchorElPopover(event.currentTarget);
+  };
+
+  const handlePopoverClose = () => {
+    setAnchorElPopover(null);
+  };
+
+  const popoverOpen = Boolean(anchorElPopover);
+
+  const handleDownloadClick = (event) => {
+    setAnchorElMenu(event.currentTarget);
+  };
+
+  const handleMenuClose = () => {
+    setAnchorElMenu(null);
+  };
+
+  const handleDownload = (format) => {
+    setAnchorElMenu(null);
+
+    const node = chartRef.current.querySelector(".apexcharts-canvas");
+    const downloadFileName = `chart.${format}`;
+
+    switch (format) {
+      case "png":
+        toPng(node)
+          .then((dataUrl) => {
+            const link = document.createElement("a");
+            link.download = downloadFileName;
+            link.href = dataUrl;
+            link.click();
+          })
+          .catch((error) => {
+            console.error("oops, something went wrong!", error);
+          });
+        break;
+      case "jpeg":
+        toJpeg(node, {
+          quality: 1,
+          backgroundColor: theme.palette.background.paper,
+        })
+          .then((dataUrl) => {
+            const link = document.createElement("a");
+            link.download = downloadFileName;
+            link.href = dataUrl;
+            link.click();
+          })
+          .catch((error) => {
+            console.error("oops, something went wrong!", error);
+          });
+        break;
+      case "svg":
+        toSvg(node)
+          .then((dataUrl) => {
+            const link = document.createElement("a");
+            link.download = downloadFileName;
+            link.href = dataUrl;
+            link.click();
+          })
+          .catch((error) => {
+            console.error("oops, something went wrong!", error);
+          });
+        break;
+      default:
+        break;
+    }
+  };
+
   const lightModePrimaryColor = useCallback(() => {
     return theme.palette.mode === "light"
       ? theme.palette.primary.light
       : theme.palette.primary.main;
   }, [theme.palette.mode, theme.palette.primary]);
 
   const darkBlueColor = useCallback(() => {
-    return theme.palette.info.main;
-  }, [theme.palette.info]);
+    return theme.palette.accent;
+  }, [theme.palette.accent]);
```

Comment:
I think `accent` is not part of the official palette, so this might break after I moved the theming.